controllers/v1/entityAssessorsController.js

```diff
@@ -84,145 +84,12 @@ module.exports = class EntityAssessors extends Abstract {
 
       try {
 
-        let programs = new Array;
```

Comment:
@aman-tunerlabs - Can you please add apidoc request-response for this API, please?

Reply:
@akash-above It is added.

Reply:
@aman-tunerlabs - Where is response?